packages/react-web/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lens-protocol/react-web",
-  "version": "0.4.1",
+  "version": "0.6.0",
```

Comment:
This one is kinda takes over from `@lens-protoco/react` for all users sung the Lens SDK on a React Web app. Seems ok to have it v0.6.0. Less opportunity for confusions.

Thoughts?

cc @desfero , @krzysu